File: LeetCode/119. Pascal's Triangle II/Solution.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class Solution {
    public List<Integer> getRow(int rowIndex) {
        int[] nums = new int[rowIndex + 1];
        nums[0] = 1;
        for (int i = 0; i < rowIndex; i++) {
            for (int j = i + 1; j > 0; j--) {
                nums[j] += nums[j - 1];
            }
        }
        List<Integer> list = new ArrayList<>();
        for (int i : nums) {
            list.add(i);
        }
        return list;
    }
}

File: LeetCode/119. Pascal's Triangle II/solution.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        ans = [1]
        for i in range(rowIndex):
            ans.append(1)
            for j in range(i, 0, -1):
                ans[j] += ans[j - 1]
        return ans

